peg-highlight/pmh_styleparser.c

```diff
@@ -817,8 +821,7 @@ static void _sty_parse(style_parser_data *p_data)
                          "No style attributes defined for style rule '%s'",
                          style_rule_name);
         
-        raw_attribute *attributes_head = NULL;
-        raw_attribute *attributes_tail = NULL;
+        
```

Comment:
Pay attention to the number of empty lines between code lines. More empty lines make no sense.